src/polysolve/LinearSolverSympiler.cpp

```diff
@@ -0,0 +1,94 @@
+#ifdef POLYSOLVE_WITH_SYMPILER
+
+#include <polysolve/LinearSolverSympiler.hpp>
+#include <sympiler/parsy/cholesky_solver.h>
+#include <Eigen/Core>
+
+namespace polysolve
+{
+class LinearSolverSympilerImpl
+{
+public:
+    LinearSolverSympilerImpl()
+        : m_solver(nullptr), m_A_csc(std::make_unique<sym_lib::parsy::CSC>()) {}
+    ~LinearSolverSympilerImpl() = default;
+
+    std::unique_ptr<sym_lib::parsy::SolverSettings> m_solver;
+    std::unique_ptr<sym_lib::parsy::CSC> m_A_csc;
+    polysolve::StiffnessMatrix m_A_copy;
```

Comment:
I am curious if we really need to store a copy of the matrix. Is this done in other solvers as well?